lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2253,6 +2253,40 @@ def encode_string(self, s, fonttype):
             return s.encode('cp1252', 'replace')
         return s.encode('utf-16be', 'replace')
 
+    def build_kern_sequence(self, string, font, fontsize,
```

Comment:
please make this private (with a leading underscore)

Reply:
That's a good point. As far as I see, they are logically equivalent. I've droped the `build_kern_sequence` stuff and relaxed the condition for the multibyte codepath.